scripts/license-header.ts

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env -S yarn node -r ts-node/register/transpile-only
+import globby, { gitignore } from "globby";
+import yargs from "yargs";
+import * as fs from "fs/promises";
+import * as Path from "path";
+
+// Globs for the files we enforce license headers for.
+const fileGlob = [
+  "ui/**/*.(svelte|ts)",
+  "native/**/*.(js|ts)",
+  "cypress/**/*.(js|ts)",
+  "public/**/*.css",
+  "!cypress/workspace",
+];
```

Comment:
What about these?

```
builder/notarize.js
ci/*
scripts/*
ui/index.html
svelte.config.js
webpack.config.ts
jest.config.js
```

Reply:
I omitted these at first because I couldn’t make automatic adding of licenses work. But we really only need automatic checking so I added these manually